File: C++ Code/Basics/Mensuration/3 D/Sphere.cpp
// WAP to calculate Volume,TSA and CSA of a Sphere.
#include<iostream>
using namespace std;
int main(){
    float r;
    cout<<"Enter the radius : ";
    cin>>r;
    cout<<"Volume = "<<(3.0/4.0)*3.14*r*r*r<<endl;
    cout<<"TSA = CSA = "<<4*3.14*r*r<<endl;
    return 0;
}

File: Java Code/Basics/Mensuration/Sphere.java
import java.util.*;

public class Sphere{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.println("For the Sphere :-");
    System.out.print("Enter radius : ");
    float r=sc.nextFloat();
    float TSA=88*r*r/7;           // TSA = 4*pie*r*r
    float V=88*r*r*r/21;         //  V = (4/3)*pie*r*r*r
    System.out.println("Total Surface Area : "+TSA);
    System.out.println("Volume : "+V);
  }
}